README.md

```diff
@@ -69,20 +73,47 @@ In der Firewall der CCU müssen je nach Anwendungsfall die Ports 2121 (HTTP), 21
 ## Bauen aus den Quellen
 
 Der CCU-Jack ist in der [Programmiersprache Go](https://golang.org/) (Version 1.15) geschrieben. Alle Distributionen des CCU-Jacks können sehr einfach und schnell auf allen möglichen Plattformen (u.a. Windows, Linux, MacOS) gebaut werden. Dafür in einem beliebigen Verzeichnis das Git-Repository klonen, oder die Quellen hinein kopieren. Danach in diesem Verzeichnis eine Kommandozeile öffnen, und folgende Befehle eingeben:
-```
+
+```bash
 cd build
 go run .
 ```
+
 In dem Hauptverzeichnis werden dann alle Distributionen gebaut.
 
 Für die Entwicklung bietet sich die Entwicklungsumgebug [Visual Studio Code](https://code.visualstudio.com/) an. Einfach das Hauptverzeichnis öffnen. Die nötigen Extensions werden automatisch zur Installation angeboten.
 
+## Docker
+
+Um ccu-jack in einem Docker Container laufen zu lassen sind folgende Schritte nötig:
+
+1. Dockerfile und ggf. docker-compose.yml von github herunterladen
+2. Docker image bauen: 
+
+  ```bash
+   export BUILD_VERSION=$(curl -Ls https://api.github.com/repos/mdzio/ccu-jack/releases/latest | grep -oP '"tag_name": "v\K(.*)(?=")')
+   
+   docker build --rm --no-cache \
+    --build-arg BUILD_VERSION="${BUILD_VERSION}" \
+    --build-arg BUILD_DATE="$(date +"%Y-%m-%dT%H:%M:%SZ")" \
+    --tag ccu-jack:latest --tag ccu-jack:${BUILD_VERSION} .
+  ```
+
+3.  a) direkt über docker laufen lassen:
+   ```docker run --rm  -v "$PWD"/ccu-jack.cfg:/app/ccu-jack.cfg:ro ccu-jack:latest```

```

Comment:
Hier muss wohl das `--rm` durch `-d` ersetzt und das `:ro` entfernt werden. Siehe auch https://github.com/mdzio/ccu-jack/pull/39#issuecomment-777803573.

Reply:
Stimmt, ist mir durchgerutscht. Passe ich ich in den Files an. 
Frage: Könnte man dem ccu-jack nicht ein config-Verzeichnis spendieren und die Certs dort speichern? das könnte man in den Container mounten, hätte die Zertifikate persistent und alles sauber aufgeräumt (oder wäre das ein eigenes Issue/Feature Request wert?)

Reply:
Der CCU-Jack liest die Konfiguration und die Zertifikate aus dem aktuellen Arbeitsverzeichnis. Die Lösung ist also ein Verzeichnis zu mounten, dorthin zu wechseln (cd) und den CCU-Jack mit einem absoluten Pfad zu starten.